README.Rmd

```diff
@@ -35,7 +35,7 @@ Features include:
 
 * A set of common methods for simply specifying data validation rules
 
-* Repeatability of data tests (avoid unintentionally breaking your dataset!)
```

Comment:
Looks like you've been pretty consistently changing dataset to data frame. This one probably makes sense as an exception, but just wanted to highlight it

Reply:
Well spotted! Yep this one was on purpose :slightly_smiling_face: I changed almost everything to data frame in the docs since it seems better to just use R terminology, but it sounded weird here so I left it as data set.